src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Internal/Controllers/InternalConnectionsController.cs

```diff
@@ -0,0 +1,217 @@
+using System.Net.Mime;
+using Altinn.AccessManagement.Api.Internal.Models;
+using Altinn.AccessManagement.Core.Constants;
+using Altinn.AccessManagement.Core.Errors;
+using Altinn.AccessManagement.Core.Models;
+using Altinn.AccessManagement.Internal.Services;
+using Altinn.AccessMgmt.Core.Models;
+using Altinn.AccessMgmt.Persistence.Core.Models;
+using Altinn.AccessMgmt.Persistence.Data;
+using Altinn.AccessMgmt.Persistence.Services;
+using Altinn.AccessMgmt.Persistence.Services.Models;
+using Altinn.Authorization.ProblemDetails;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement.Mvc;
+
+namespace Altinn.AccessManagement.Api.Internal.Controllers;
+
+/// <summary>
+/// Controller for en user api operations for connections
+/// </summary>
+[ApiController]
+[Route("accessmanagement/api/v1/internal/connections")]
+[FeatureGate(AccessManagementInternalFeatureFlags.ControllerConnections)]
+[Authorize(Policy = AuthzConstants.SCOPE_PORTAL_ENDUSER)]
+public class InternalConnectionsController(IInternalConnectionService connectionService) : ControllerBase
+{
+    private IInternalConnectionService ConnectionService { get; } = connectionService;
+
+    /// <summary>
+    /// Get connections between the authenticated user's selected party and the specified target party.
+    /// </summary>
+    [HttpGet]
+    [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_ENDUSER_READ)]
+    [ProducesResponseType<PaginatedResult<CompactRelationDto>>(StatusCodes.Status200OK, MediaTypeNames.Application.Json)]
+    [ProducesResponseType<AltinnProblemDetails>(StatusCodes.Status400BadRequest, MediaTypeNames.Application.Json)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    public async Task<IActionResult> GetConnections([FromQuery] ConnectionInput connection, [FromQuery, FromHeader] PagingInput paging, CancellationToken cancellationToken = default)
```

Comment:
Disse endepunktene burde kanskje skrives om fra å bruke ConnectionInput med Party, From og To som er laget for å støtte 2-veis oppslag og operasjoner for både from og to. 

Disse operasjonene for å legge til systembruker som rettighetshaver og delegere pakker til denne skal bare kunne gjøres som tilgangsstyrer for From-party.

Så i stedet for ConnectionInput, er trenger vi obligatoriske parametere:
`?party={fromPartyUuid}&to={systemUserUuid}`

Reply:
evt. om det er enklere for å gjenbruke service implementasjon så må vi hvertfall tvinge at party og from er like i alle requests.

?party={fromPartyUuid}&from={fromPartyUuid}&to={systemUserUuid}